src/odemis/gui/util/img.py

```diff
@@ -1611,9 +1616,16 @@ def get_ordered_images(streams, raw=False):
         # FluoStreams are merged using the "Screen" method that handles colour
         # merging without decreasing the intensity.
         if not raw:
-            data = s.image.value
+            if isinstance(s.raw, list):
```

Comment:
Don't check .raw to know what to do with data. => Check if data is a 2D tuple, and if so, call mergeTiles().